compositor_pipeline/src/pipeline.rs

```diff
@@ -314,7 +314,7 @@ impl Pipeline {
             return;
         }
         info!("Starting pipeline.");
-        let (frames_sender, frames_receiver) = bounded(20);
+        let (frames_sender, frames_receiver) = bounded(1);
```

Comment:
What about audio?

Reply:
audio is processed in sync  with video, so it will never buffer too much. It might be good to reconsider when audio support is finished, but for now I don't see point in speculating what should we do

Reply:
Then why do those channels have different capacities?

Reply:
because sample rate and distance between frame is different, plus it does not hurt us to keep more audio

Reply:
also initially audio and video output was not synced, it was necessary to implement event scheduling